packages/server/modules/core/loaders.ts

```diff
@@ -167,531 +63,67 @@ const buildDataLoaderCreator = (selfClearing = false) => {
  * Build request-scoped dataloaders
  * @param ctx GraphQL context w/o loaders
  */
-export function buildRequestLoaders(
+export async function buildRequestLoaders(
   ctx: AuthContext,
   options?: Partial<{ cleanLoadersEarly: boolean }>
 ) {
-  const userId = ctx.userId
-
   const createLoader = buildDataLoaderCreator(options?.cleanLoadersEarly || false)
   const modulesLoaders = graphDataloadersBuilders()
 
-  const loaders = {
-    ...(Object.assign(
-      {},
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-      ...modulesLoaders.map((l) => l({ ctx, createLoader }))
-    ) as Record<string, unknown>),
-    streams: {
```

Comment:
Moved all of the `core` dataloaders to their own module so that this file is only about dataloader bootstrapping